source/administration/object-management/data-compression.rst

```diff
@@ -0,0 +1,191 @@
+
+- Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+Existing uncompressed objects are not modified.
+To configure which extensions and types to compress, see :ref:`minio-data-compression-configure-objects`.
+
+To view the current compression settings:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression
+
+
+Disable Data Compression
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To disable data compression, use :mc-cmd:`mc admin config set` to set the :mc-conf:`compression` key :mc-conf:`~compression.enable` option to ``off``:
+
+The following disables data compression for new objects:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression enable=off
+
+- Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+Existing compressed objects are not modified.
+
+.. _minio-data-compression-configure-objects:
+
+Configure Which Objects to Compress
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configure the objects to compress by specifying the desired file extensions and media types in :mc-conf:`~compression.extensions` or :mc-conf:`~compression.mime_types` arguments.
+
+By default, MinIO compresses the following extensions and types:
+
+* File extensions: ``.txt``, ``.log``, ``.csv``, ``.json``, ``.tar``, ``.xml``, ``.bin``
+* Media types: ``text/*``, ``application/json``, ``application/xml``, ``binary/octet-stream``
+
+.. admonition:: Default excluded extensions and types are never compressed
+   :class: note
+
+   Some objects cannot be efficiently compressed.
+   MinIO will not attempt to compress these objects, even if they are specified in :mc-conf:`~compression.extensions` or :mc-conf:`~compression.mime_types` arguments.
+
+The sections below describe how to configure compression for the desired file extensions and media types.
+
+Compress All Compressible Objects
++++++++++++++++++++++++++++++++++
+
+To compress all objects except the default excluded ones, use :mc-cmd:`mc admin config set` to set the :mc-conf:`compression` key :mc-conf:`~compression.extensions` and :mc-conf:`~compression.mime_types` options to empty lists.
+
+The following compresses all objects, except the default incompressible objects:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression extensions= mime_types=
+
+- Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+  
+Compress Objects by File Extension
+++++++++++++++++++++++++++++++++++
+
+To compress objects with certain file extensions, use :mc-cmd:`mc admin config set` to set the :mc-conf:`compression` key :mc-conf:`~compression.extensions` option to a list of the desired file extensions.
+
+The following command compresses files with the extensions ``.bin`` and ``.txt``:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression extensions=".bin, .txt"
```

Comment:
I tested this and having the space is ok. The original examples from the readme omit whitespace between items. Unless there is a concern, going with space for readability.